pkg/virt-operator/creation/components/deployments.go

```diff
@@ -246,18 +246,21 @@ func newBaseDeployment(name string, namespace string, repository string, version
 func newPodAntiAffinity(key, topologyKey string, operator metav1.LabelSelectorOperator, values []string) *corev1.Affinity {
 	return &corev1.Affinity{
 		PodAntiAffinity: &corev1.PodAntiAffinity{
-			RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+			PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
 				{
-					LabelSelector: &metav1.LabelSelector{
-						MatchExpressions: []metav1.LabelSelectorRequirement{
-							{
-								Key:      key,
-								Operator: operator,
-								Values:   values,
+					Weight: 1,
```

Comment:
@rmohr, @davidvossel i am not sure if this is correct number, which should be here? In kubernetes doc (https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature), they use 1 as example.

Reply:
in our case, I believe anything > 0 will suffice.